### ISSUE TITLE
FEATURE: 

### ISSUE THREAD

**Issue by tketchem14** [northeastern-rc/rc-public-documentation#99]
### Documentation request

The Getting help page needs to be updated to correct references to the IT Statuspage.

### Suggested fix for documentation

Heading should be IT Statuspage, and the paragraph needs to be corrected as well. (I already fixed this up in the maintenance section I'm writing up right now, and I think that will be a better location for the information anyway, but I'm just submitting a test request) 

### Problem Description (if any)

Does not conform to ITS Comms style guide. 

### Describe alternatives you've considered

N/A

### Additional context

<img width="380" alt="image" src="https://github.com/northeastern-rc/rc-public-documentation/assets/101415920/2be178de-0036-4367-8de7-e75868458718">

**Comment by visionjo** [comment]
@tketchem14, does it look okay to you?

**Comment by tketchem14** [comment]
@visionjo, yeah, it looks good, easy to fill out, easy to add a picture. If it's easy enough for me, it should be easy enough for our users!
